src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/request/AvailabilityRequest.kt

```diff
@@ -0,0 +1,120 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.model.request
+
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.media.Schema.RequiredMode
+import jakarta.validation.Constraint
+import jakarta.validation.ConstraintValidator
+import jakarta.validation.ConstraintValidatorContext
+import jakarta.validation.Payload
+import jakarta.validation.constraints.NotEmpty
+import java.time.Duration
+import java.time.LocalDate
+import java.time.LocalTime
+import kotlin.reflect.KClass
+
+@Schema(description = "The request object sent to the availability check endpoint")
+data class AvailabilityRequest(
+  @Schema(
+    description = "The booking type",
+    example = "COURT",
+    requiredMode = RequiredMode.REQUIRED,
+  )
+  val bookingType: BookingType,
```

Comment:
Not picking fault here but I am keen to make sure we are consistent with required attrs.  If this works then I am all good, all model objects should follow the same pattern.

Historically we have used a ? mark even on required fields.

Reply:
It had a `required = true` on the schema, but this shows up as deprecated now and suggests the RequiredMode.REQUIRED as the current way.

Reply:
Made it BookingType? optional now.